airflow/docker-compose.yaml

```diff
@@ -54,14 +54,16 @@ x-airflow-common:
     AIRFLOW__CORE__PLUGINS_FOLDER: /opt/airflow/gcs/plugins
     AIRFLOW__WEBSERVER__RELOAD_ON_PLUGIN_CHANGE: 'true'
     CALITP_BQ_MAX_BYTES: 10000000000
-    # connections
-    # see https://stackoverflow.com/a/55064944/1144523
-    AIRFLOW_CONN_GOOGLE_CLOUD_DEFAULT: "google-cloud-platform://:@:?extra__google_cloud_platform__project=cal-itp-data-infra"
```

Comment:
**question:** This is just clean-up, right? I.e., this change has nothing to do with this PR, but we aren't using this Airflow connection anywhere, correct?